common/religion/doctrines/00_core_tenets.txt

```diff
@@ -3896,4 +3901,42 @@
 			cannibalism_legal = yes
 		}
 	}
-}
+
+	#Living Divinity, basically Warrior Monks and Unrelenting Faith
+	tenet_living_divinity = {
+		icon = wc_core_tenet_flexing_muscles
```

Comment:
Find maybe something else

Reply:
I did ask for something in the 2d art channel, just kinda waiting on that